.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
Yea, that works for me. I noticed they have a feature request to support `editorconfig`. Hopefully soon.